quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/dist/approvals/cli/help/HelpCommandDistTest.testStartDevHelp.windows.approved.txt

```diff
@@ -165,6 +149,11 @@ HTTP(S):
 --http-max-queued-requests <requests>
                      Maximum number of queued HTTP requests. Use this to shed load in an overload
                        situation. Excess requests will return a "503 Server not Available" response.
+--http-pool-max-threads <threads>

```

Comment:
This seems unrelated to this PR, right?

Reply:
@mabartos that was added as a part of the https://github.com/keycloak/keycloak/pull/26659 PR.